documentation/getting-started/getting-started-netbeans.asciidoc

```diff
@@ -0,0 +1,96 @@
+---
+title: Starting a Vaadin Project in NetBeans
```

Comment:
All other docs only mention Eclipse and IntelliJ, when we have this available too. Is this by choice? I wouldn't disagree with that, but some may find it confusing.